src/globus_sdk/_testing/data/groups/_common.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any, Dict
+
+GROUP_ID = "d3974728-6458-11e4-b72d-123139141556"
+
+BASE_GROUP_DOC: Dict[str, Any] = {
+    "name": "Claptrap's Rough Riders",
+    "description": "No stairs allowed.",
```

Comment:
Your reference has not gone unappreciated :)

Reply:
That warms my frosty heart. This is a real group in production, which I created when the reference was very fresh. 😸